lib/log-formatters.js

```diff
@@ -30,7 +30,10 @@ module.exports = function (d) {
    * Format metadata
    */
   d.formatters.m = md => {
-    return humanID(md)
+    if (md instanceof ao.MB) {
+      return md.toString(1);
```

Comment:
at this point that's an idiom for me but not for everyone so it might be nicer to use a constant. am using the already existing `MB.fmtHuman` (most constants in node are not LOUD_BIG_UNDERSCORE_SEPAPATED_THINGS).